iocore/net/P_UnixNet.h

```diff
@@ -389,30 +400,171 @@ check_transient_accept_error(int res)
   }
 }
 
+static inline void
+net_activity(UnixNetVConnection *vc, EThread *thread)
+{
+  Debug("socket", "net_activity updating inactivity %" PRId64 ", NetVC=%p", vc->inactivity_timeout_in, vc);
+  (void)thread;
```

Comment:
I also think `net_activity` was better as a method of `NetVConnection` or `UnixNetVConnection` as it always requires a instance of that class.